### ISSUE TITLE
Scratch Org features missing specified in new format in project-scratch-def.json

### ISSUE THREAD

**Issue by ashar165** [forcedotcom/salesforcedx-vscode#709]
### Summary

Winter 19 release of Salesforce came with new format of scratch org definition file. It can be read here: [Scratch Org Definition Configuration Values](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_scratch_orgs_def_file_config_values.htm)
This format accepts different settings in a format with true or false value provided for setting. Earlier, it used to be two arrays in json, one was enabled and another was disabled. It used to work great. But this new format seems to buggy and does not work when Omni-Channel settings are provided true with other 2+ settings. When only two settings are specified, it works fine, but when more than 2 settings are specified in json, it does not work for settings like, Omni-Channel, Live-Agent etc.

### Steps To Reproduce:

1. Modify project-scratch-def.json file on your local machine before creating a new scratch org. JSON should have at-least 3 Org settings specified as true as given below:
```
{
    "orgName": "sfdx sira",
    "country": "AU",
    "edition": "Developer",
    "description": "sfdx sira",
    "hasSampleData": "false",
    "features": ["Communities",
        "LightningSalesConsole",
        "ForceComPlatform",
        "API"
    ],
	"settings": {
		"nameSettings" :{
			"enableNameSuffix":false
		},
		"liveAgentSettings" : {
			"enableLiveAgent":true
		},
		"entitlementSettings" : {
			"enableEntitlements":true
		},
		"accountSettings" : {
			"enableAccountTeams":true,
			"showViewHierarchyLink":true
		},
		"opportunitySettings" : {
			"enableOpportunityTeam":true
		},
		"activitiesSettings" : {
			"enableActivityReminders":true
		},
		"omniChannelSettings": {
			"enableOmniChannel": true
		},
	}
}

```
1.  Run sfdx force:org:create command to create scratch org.
1. Open the scratch org and verify if Omni-Channel has been enabled.

### Expected result

Omni-Channel, Live-Agent and other specified settings should be enabled.

### Actual result

Omni-Channel and Live-Agent settings aren't enabled.

**VS Code Version**: 1.26.1

**SFDX CLI Version**: 6.36.0-391bf95737 (windows-x64) node-v8.9.4

**OS and version**: Win10 Pro(64 bit) Version-1703

**Internal Work Item**: [W-5626597](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000005oVh8IAE/view)

**Comment by ntotten** [comment]
@ashar165 Are you able to open a support case? This isn't actually an issue with VS Code. I passed it along to the right teams, but it will be most productive to have it be a support case from a customer.

**Comment by ntotten** [comment]
It looks like this might be a typo:

```
"omniChannelSettings": {
   "enableOmniChannel": truea
},
```

the value for `truea` needs to be `true`

**Comment by ntotten** [comment]
Actually, after testing without the typo it still is causing the error. We are investigating.

**Comment by ashar165** [comment]
Sorry for the typo. I'll correct it.

**Comment by lcampos** [comment]
Update: The majority of the issues with settings have been fixed and released on the server side. 

@ashar165 I've tested with your initial configuration and everything was configured properly, let me know if you still see issues with the settings.

There's a fix that still needs to be worked on for `accountSettings` and is being tracked internally in W-5626597. There's also an ongoing conversation about this in the success community (https://success.salesforce.com/0D53A00003nTXLV).

cc @ntotten
